jboss-eap-standalone-openshift/azuredeploy.parameters.json

```diff
@@ -0,0 +1,25 @@
+ {
+   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+          "value": "changeme"
+        },
+        "adminPassword": {
```

Comment:
remove this

Reply:
This is the password we use for the OCP console. We need this password to be specified by the user.

Reply:
no, this used to be the vm password that was never used because we're using ssh authn

Reply:
I have changed the name of this parameter to openshiftPassword. We use this to set up the password of the OpenShift Web Console.

Reply:
There is no longer an adminPassword in the parameters list because it's never used. There is a parameter "openshiftPassword" now that's used to set up the password for the OpenShift web console.